### ISSUE TITLE
alert for heat/cool differential - incorrect detected value from Ecobee

### ISSUE THREAD

**Issue by fermulator** [beestat/app#260]
Great project!

My first alerts were:
![2020-03-01_13:46:45](https://user-images.githubusercontent.com/6778269/75631729-4e61bd00-5bc3-11ea-8ff8-7de775eeb8a9.png)

And I drill into the heat/cool differential one:
![2020-03-01_13:46:52](https://user-images.githubusercontent.com/6778269/75631730-54f03480-5bc3-11ea-92da-b586966dc114.png)

So first off, probably we should add a link to the support article: https://support.ecobee.com/hc/en-us/articles/360002020552-Threshold-settings-for-ecobee-thermostats
(not all users would be aware of this setting, and it would be helpful to give them a link to learn more)

Second, I was surprised by this at all. I barely remembered it had a setting, but was fairly confident when I first set up my thermostat I would've researched and tweaked all the settings.  A 0.3oC differential seems tiny!

Lo-and-behold, I was right, I have already set mine to 2.8oC.
![ecobee3_thermostat_delta_temp_threshold](https://user-images.githubusercontent.com/6778269/75631814-160eae80-5bc4-11ea-80c0-cf116cd7cc96.JPG)

fwiw, I never use auto-mode anyway, so this doesn't affect me, but I suppose this is a bug in that beestats has incorrectly determined the Ecobee's temperature threshold for my device.

**Comment by fermulator** [comment]
@ziebelje  - any thoughts on ^^?

**Comment by ziebelje** [comment]
Nope...but I've been knee-deep in some other cool updates (coming soon!) so I haven't really looked at it yet.
